pkg/catalog/outbound_traffic_policies.go

```diff
@@ -62,8 +62,9 @@ func (mc *MeshCatalog) listOutboundTrafficPoliciesForTrafficSplits(sourceNamespa
 	apexServices := mapset.NewSet()
 	for _, split := range mc.meshSpec.ListTrafficSplits() {
 		svc := service.MeshService{
-			Name:      kubernetes.GetServiceFromHostname(split.Spec.Service),
-			Namespace: split.Namespace,
+			Name:        kubernetes.GetServiceFromHostname(split.Spec.Service),
+			Namespace:   split.Namespace,
+			ClusterName: "local",
```

Comment:
can we add this to the constants file, and say constants.LocalClusterID?

Reply:
This doesn't seem like the cluster name, rather the cluster's trust domain suffix. If this is a special notation for denoting local clusters without their actual name, should move this to `pkg/constants`. If this is not meant to represent the name of the cluster, the field needs to be renamed as well.

Reply:
ya that's true. Thoughts on naming in that case? Alternatively, We could optionally put the cluster domain, ie: cluster.local, and call it ClusterDomain.

Reply:
If `ClusterDomain` works with the proposed approach, that would be the best way forward, given we are using a similar concept for defining identities (search `ClusterLocalTrustDomain`)

Reply:
thinking about this a bit more, it would be cleaner if we can set this to just "local". There's places where we will call ms.Name(), where we want it to only have the "local".

@shashankram  is there a term we can use to refer to this? Conceptually it is the name of a cluster in an OSM MultiCluster setup. That is to say, it is not stricly tied to kubernetes. In fact a name of "cluster-x" might not be "cluster.cluster-x" in a multi cluster setup on VM's or bare metal.

Cluster is a super overloaded term... but what do we want to call a cluster in an OSM multi cluster scenario? Whatever we call that, should be the name of this field.

Reply:
@steeling I don't have a strong opinion on this. My comment was merely pointing out that `Cluster == local` is not the cluster's name, and if we end up using the trust domain instead then we should name the field a such. 

I am with whatever you and @snehachhabria decide.

Reply:
I agree with @shashankram, calling that field the `ClusterName` seems inappropriate since it is representing the domain